_projects/2023/100061715/100061715.Rmd

```diff
@@ -0,0 +1,256 @@
+---
+title: "Unveiling birth clinical practice in Spain: mapping the cesarian sections and birth dynamics"
+description: "Exploring the nuances of maternal healthcare in Spain through an augmented map, showcasing original cesarean section rates alongside additional insights such as total births per autonomous community and cesarean rates adjusted for the average maternal age."
+categories: 2023
+author: "Celia Muñoz"
+date: "`r Sys.Date()`"
+---
+
+```{r}
+# Loading packages
+library(tidyverse)
+library(mapSpain)
+library(readr)
+library(ggplot2)
+library(sf)
+library(stringdist)
+library(fuzzyjoin)
+library(ggtext)
+```
+
+
+```{r}
+# Adjusting locale settings
+my_locale <- locale(decimal_mark = ",", grouping_mark = ".")
+
+
+# Reading data
+cesareas <- read_delim("tasa_cesareas.csv", 
+                 skip = 10, 
+                 delim = ";",
+                 col_names = c("ccaa", "n_partos", "estancia_partos", "edad_partos", "n_cesareas", "edad_cesareas", "estancia_cesareas", "tasa_cesareas"),
```

Comment:
Very long lines of code get out of the page. Please restyle all code so that everything fits within the column width. A maximum width of 80 characters is generally advisable.